src/SFA.DAS.AdminService.Web/Views/Roatp/Apply/Gateway/Pages/SubcontractorDeclaration.cshtml

```diff
@@ -87,25 +55,25 @@
                     }
                 </tbody>
             </table>
-
         </div>
     </div>
-
     @if (Model.ApplicationId != Guid.Empty)
     {
         <div class="govuk-grid-row">
             <div class="govuk-grid-column-two-thirds">
                 <form class="js-disable-on-submit" asp-action="EvaluateSubcontractorDeclarationPage" asp-controller="RoatpGatewayExperienceAndAccreditation">
-                    <input type="hidden" name="ApplicationId" value="@Model.ApplicationId" />
-                    <input type="hidden" name="PageId" value="@Model.PageId" />
-                    <input type="hidden" name="ApplicationSubmittedOn" value="@Model.ApplicationSubmittedOn" />
-                    <input type="hidden" name="Ukprn" value="@Model.Ukprn" />
-                    <input type="hidden" name="SourcesCheckedOn" value="@Model.SourcesCheckedOn" />
-                    <input type="hidden" name="HasDeliveredTrainingAsSubcontractor" value="@Model.HasDeliveredTrainingAsSubcontractor" />
-                    <input type="hidden" name="ContractFileName" value="@Model.ContractFileName" />
-
+                    @Html.HiddenFor(x => x.ApplicationId)
```

Comment:
Use of deprecated HTML Helper functions. Use tag helpers

Reply:
done